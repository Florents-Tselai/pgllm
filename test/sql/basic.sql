-- testing a simple model coded in C and store in the static catalog
select llm_generate('hello', 'repeat-3');
