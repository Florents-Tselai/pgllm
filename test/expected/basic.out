-- testing a simple model coded in C and store in the static catalog
select llm_generate('hello', 'repeat-3');
  llm_generate   
-----------------
 hellohellohello
(1 row)

